apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -97,34 +99,44 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
   const isLoading = isLoadingEnvironment || isUpdating;
 
   return (
-    <Modal opened={isOpen} title={<Title variant="section">Update Novu Endpoint URL</Title>} onClose={toggleOpen}>
-      <Box colorPalette={'mode.local'}>
-        <Input
-          label={'Novu Endpoint URL'}
-          description={
-            'This url should be a full URL to the Novu Endpoint including the /api/novu path, e.g. https://your.api.com/api/novu'
-          }
-          onChange={onBridgeUrlChange}
-          value={url}
-          disabled={isLoading}
-          variant="preventLayoutShift"
-          error={urlError}
-          className={css({ marginBottom: '16px' })}
-        />
-        <HStack justify={'space-between'}>
-          <DocsButton
-            TriggerButton={({ onClick }) => (
-              <button onClick={onClick} className={hstack({ gap: 'margins.icons.Icon20-txt', cursor: 'pointer' })}>
-                <IconOutlineMenuBook />
-                <Text variant="secondary">Learn more in the docs</Text>
-              </button>
-            )}
+    <>
+      <Modal opened={isOpen} title={<Title variant="section">Update Novu Endpoint URL</Title>} onClose={toggleOpen}>
+        <Box colorPalette={'mode.local'}>
+          <Input
+            label={'Novu Endpoint URL'}
+            description={
+              'This url should be a full URL to the Novu Endpoint including the /api/novu path, e.g. https://your.api.com/api/novu'
+            }
+            onChange={onBridgeUrlChange}
+            value={url}
+            disabled={isLoading}
+            variant="preventLayoutShift"
+            error={urlError}
+            className={css({ marginBottom: '16px' })}
           />
-          <Button size={'md'} loading={isLoading} onClick={onUpdateClick}>
-            Update
-          </Button>
-        </HStack>
-      </Box>
-    </Modal>
+          <HStack justify={'space-between'}>
+            <div>
+              <HStack gap="50" className={css({ color: 'typography.text.secondary' })}>
+                <IconOutlineMenuBook />
+                <LinkText
+                  onClick={(e) => {
+                    e.preventDefault();
+                    setPath('concepts/endpoint');
+                    toggle();
+                  }}
+                  href=""
+                >
+                  Learn more in our docs
+                </LinkText>
+              </HStack>
+            </div>
```

Comment:
```suggestion
              <HStack gap="50" className={css({ color: 'typography.text.secondary' })}>
                <IconOutlineMenuBook />
                <LinkText
                  onClick={(e) => {
                    e.preventDefault();
                    setPath('concepts/endpoint');
                    toggle();
                  }}
                  href=""
                >
                  Learn more in our docs
                </LinkText>
              </HStack>
```
(Please remove the div)